### DISCUSSION TITLE
unsaved changes? VSCode for the Web and FileSystemProvider

### DISCUSSION THREAD

**Discussion by KernelDeimos** [microsoft/vscode-discussions#335] [Extension Development QnA] [answered]
Hello! I'm working on an extension for VS Code for the Web that enables access to a filesystem on a remote service. What I noticed is that VS Code for the Web does not have any notion of "unsaved" files. I'm wondering if it's possible to get this behavior back. If I can have VS Code for the Web show a notice if a file was modified while being edited - just like the native desktop application - that would be even better.

**Comment by r3m0t** [comment, marked as answer]
Which notice do you mean from the desktop application?

Did you implement `stat` to update the `mtime` and `size` when the file is modified in the remote service? And are you firing `onDidChangeFile` when the file is modified in the remote service?
